src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -26,8 +26,10 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
   setPendingForm
 }) => {
   const [ tabIndex, setTabIndex ] = React.useState(0);
-  const [ disabledValue, setdisabledValue ] = React.useState<boolean>();
-  const [ requiredField, setRequiredField ] = React.useState<any>();
+  const [ disabledValue, setdisabledValue ] = React.useState<any>();
+  const [ requiredField, setRequiredField ] = React.useState<any>("");
+  const [ requiredFieldBoolean, setRequiredFieldBoolean ] = React.useState<string>("");
```

Comment:
useState type should be something else than any.
Also disabledValue and requiredFieldBoolean could be named more descriptively.